<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>helpDialog</class>
 <widget class="QDialog" name="helpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1037</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(91,90,90);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameMonitorBar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#008F96;border-width: 0;
border-radius : 0;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox_19">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>580</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Berlin Sans FB</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox::title{
	color : white;
	left: 10px;
	margin-left: 0px;
	padding: 0 1px;
}

QGroupBox{
	border-width: 5px;
	border-style: solid;
	border-color: white;
	margin-top: 0.5ex;
	border-radius: 5px;
}</string>
       </property>
       <property name="title">
        <string>Team Member</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_64">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>30</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_32">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_57">
            <item>
             <widget class="QGroupBox" name="groupBox_20">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QGroupBox::title{
	color : white;
	left: 10px;
	margin-left: 0px;
	padding: 0 1px;
}

QGroupBox{
	border-width: 3px;
	border-style: solid;
	border-color: white;
	margin-top: 0.5ex;
	border-radius: 5px;
	background: rgb(0,0,0,0);
}</string>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_65">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QFrame" name="frame_48">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>120</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>120</width>
                   <height>130</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame
{
   border-image: url(:/icons/icos/MaHaotian.jpg) 0 0 0 0 stretch stretch;
   border-width: 0px;
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_33">
              <item>
               <widget class="QLabel" name="label_72">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>65</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Berlin Sans FB</family>
                  <pointsize>26</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color : white;</string>
                </property>
                <property name="text">
                 <string>Ma Haotian</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_73">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Berlin Sans FB</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color : white;</string>
                </property>
                <property name="text">
                 <string>118010220</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_58">
            <item>
             <widget class="QGroupBox" name="groupBox_21">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>130</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QGroupBox::title{
	color : white;
	left: 10px;
	margin-left: 0px;
	padding: 0 1px;
}

QGroupBox{
	border-width: 3px;
	border-style: solid;
	border-color: white;
	margin-top: 0.5ex;
	border-radius: 5px;
	background: rgb(0,0,0,0);
}</string>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_66">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QFrame" name="frame_49">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>120</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame
{
   border-image: url(:/icons/icos/MaoYu.jpg) 0 0 0 0 stretch stretch;
   border-width: 0px;
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_34">
              <item>
               <widget class="QLabel" name="label_74">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>65</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Berlin Sans FB</family>
                  <pointsize>26</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color : white;</string>
                </property>
                <property name="text">
                 <string>Mao Yu</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_75">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Berlin Sans FB</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color : white;</string>
                </property>
                <property name="text">
                 <string>118010224</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_59">
            <item>
             <widget class="QGroupBox" name="groupBox_22">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>130</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QGroupBox::title{
	color : white;
	left: 10px;
	margin-left: 0px;
	padding: 0 1px;
}

QGroupBox{
	border-width: 3px;
	border-style: solid;
	border-color: white;
	margin-top: 0.5ex;
	border-radius: 5px;
	background: rgb(0,0,0,0);
}</string>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_67">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QFrame" name="frame_50">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>120</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame
{
   border-image: url(:/icons/icos/WuWei.jpg) 0 0 0 0 stretch stretch;
   border-width: 0px;
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_35">
              <item>
               <widget class="QLabel" name="label_76">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>65</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Berlin Sans FB</family>
                  <pointsize>26</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color : white;</string>
                </property>
                <property name="text">
                 <string>Wu Wei</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_77">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Berlin Sans FB</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color : white;</string>
                </property>
                <property name="text">
                 <string>118010335</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_60">
            <item>
             <widget class="QGroupBox" name="groupBox_23">
              <property name="minimumSize">
               <size>
                <width>125</width>
                <height>125</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>125</width>
                <height>130</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QGroupBox::title{
	color : white;
	left: 10px;
	margin-left: 0px;
	padding: 0 1px;
}

QGroupBox{
	border-width: 3px;
	border-style: solid;
	border-color: white;
	margin-top: 0.5ex;
	border-radius: 5px;
	background: rgb(0,0,0,0);
}</string>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_68">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QFrame" name="frame_51">
                 <property name="minimumSize">
                  <size>
                   <width>120</width>
                   <height>120</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QFrame
{
   border-image: url(:/icons/icos/ZhangShiqi.jpg) 0 0 0 0 stretch stretch;
   border-width: 0px;
}</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_36">
              <item>
               <widget class="QLabel" name="label_78">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>65</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Berlin Sans FB</family>
                  <pointsize>26</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color : white;</string>
                </property>
                <property name="text">
                 <string>Zhang Shiqi</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_79">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>60</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Berlin Sans FB</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color : white;</string>
                </property>
                <property name="text">
                 <string>118010416</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="font">
        <font>
         <family>Berlin Sans FB</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    color:rgb(255,255,255);
}

QCheckBox::indicator {
    width: 10px;
    height: 10px;
}

QCheckBox::indicator:unchecked {
    border:2px solid rgb(51,51,51);
	background:rgb(91,90,90);
}

QCheckBox::indicator:unchecked:pressed {
	border:2px solid rgb(51,51,51);
   	background:rgb(0,143,170);
}

QCheckBox::indicator:checked {
	background-color:rgb(0,143,170);
    border: 2px solid rgb(51,51,51);
}

QCheckBox::indicator:checked:pressed {
    border:2px solid rgb(51,51,51);
	background:rgb(91,90,90);
}

</string>
       </property>
       <property name="text">
        <string>Donate 5 mins every 4 hours</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonFinish">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: none;
	background-color: rgba(0,0,0,0);
	color : white;
}
QPushButton:hover {
	background-color: rgb(51,51,51);
}
QPushButton:pressed {	
	background-color: #7A0000;
}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_43">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>15</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(51,51,51,220);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
